README.md

```diff
@@ -588,7 +588,7 @@ Other Style Guides
     }
 
     // good
-    const foo = function bar() {
+    const foo = function foo () {
```

Comment:
Thanks. Is there a rule in the style guide that explains the part about using a different lexically bound name reference? I haven't come across that before...